client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -71,10 +71,17 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep, flow }: S
 
 	const { title, launchTitle, subtitle } = getLaunchpadTranslations( flow );
 
+	const goals = useSelect(
+		( select ) => ( select( ONBOARD_STORE ) as OnboardSelect ).getGoals(),
+		[]
+	);
+	const hasGoalPaidSubscribers = goals?.includes( 'paid-subscribers' );
```

Comment:
We can import the key from `@automattic/data-stores` similar to how we're doing it at `newsletter-setup/index.ts` (https://github.com/Automattic/wp-calypso/pull/76386/files#diff-ac3cace53b8a860d7deeccd0cd5c7ba206b8447549b035e5daf4aecee1e2516dR98). That way we ensure it's always the same.

Reply:
Done